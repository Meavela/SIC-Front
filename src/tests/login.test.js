import React from 'react';
import renderer from 'react-test-renderer';
import Login from '../components/login';

it('login page correctly', () => {
    const tree = renderer
      .create(<Login />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });