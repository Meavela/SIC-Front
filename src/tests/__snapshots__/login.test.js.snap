// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`login page correctly 1`] = `
<div
  className="loginContent"
>
  <form
    className=""
  >
    <div
      className="form-group"
    >
      <label
        className="form-label"
        htmlFor="formBasicUsername"
      >
        Username :
      </label>
      <input
        className="form-control"
        id="formBasicUsername"
        onChange={[Function]}
        placeholder="Enter username"
        type="text"
      />
      <small
        className="text-muted form-text"
      />
    </div>
    <div
      className="form-group"
    >
      <label
        className="form-label"
        htmlFor="formBasicPassword"
      >
        Password :
      </label>
      <input
        className="form-control"
        id="formBasicPassword"
        onChange={[Function]}
        placeholder="Enter Password"
        type="password"
      />
    </div>
    <div
      className="form-group"
    >
      <label
        className="errorContent form-label"
      >
        
      </label>
    </div>
    <button
      className="btn btn-primary"
      disabled={false}
      onClick={[Function]}
      type="button"
    >
      Login
    </button>
  </form>
</div>
`;
